docker-compose.yml

```diff
@@ -1,37 +1,51 @@
 ---
-version: "3.7"
-
-# This docker-compose file is used to build and test the container
-
-secrets:
-  quote_txt:
-    file: ./src/secrets/quote.txt
-
+version: '3'
 services:
-  example:
-    # Run the container normally
+
+  li-pca-api:
     build:
-      # VERSION must be specified on the command line:
-      # e.g., --build-arg VERSION=0.0.1
       context: .
-      dockerfile: Dockerfile
-    image: cisagov/example
-    init: true
-    restart: "no"
-    environment:
-      - ECHO_MESSAGE=Hello World from docker-compose!
+      dockerfile: ./docker/api/Dockerfile
+    stdin_open: true
+    tty: true
+    container_name: li-pca-api
+    hostname: li-pca-api
+    restart: on-failure
+    links:
```

Comment:
What are you using `links` to do? If you just need to require the `mongodb` service then you should use `depends_on`. You are already using a custom network configuration. Per the warning [here](https://docs.docker.com/compose/compose-file/compose-file-v3/#links) we should avoid using deprecated functionality.

Reply:
Yep it should go away and add depends_on. I must have added it as an old habit. Addressed in 597798f